App/routes/components/ProjectMenu.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { View, Image, StyleSheet } from 'react-native'
+import {
+  Menu,
+  MenuOptions,
+  MenuOption,
+  MenuTrigger,
+} from 'react-native-popup-menu';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCurrentProjectUuid, getCurrentProject, renameProject } from '../../state/projects';
+import RenameProjectModal from './RenameProjectModal'
+
+
+export default () => {
+  const dispatch = useDispatch()
+  const [modalVisible, setModalVisible] = useState(false)
```

Comment:
nice habbit to have is to name boolean variables starting with is
isModalVisible
in non-strict languages like js it helps to easily understand that this varialbe contains boolean